services/database/relational/features.yaml

```diff
@@ -1,7 +1,3 @@
-title: CCC Relational Database Management System Features
-category_id: CCC.RDMS
-type: Databases
-description: This documents the minimual set of features that should be present for a RDMS service to be considered for use in financial services ecosystems.
 features:
```

Comment:
we need `common_features` section before `features`

Reply:
```suggestion
common_features: []
features:
```